core/src/test/java/io/confluent/rest/KafkaExceptionMapperTest.java

```diff
@@ -133,8 +134,13 @@ public void testKafkaExceptions() {
     //test couple of kafka exception
     verifyMapperResponse(new CommitFailedException(), Status.INTERNAL_SERVER_ERROR,
         KAFKA_ERROR_ERROR_CODE);
-    verifyMapperResponse(new ConcurrentTransactionsException("some message"), Status.INTERNAL_SERVER_ERROR,
-        KAFKA_ERROR_ERROR_CODE);
+
+    Exception cte = new ConcurrentTransactionsException("some message");
+    if(cte instanceof RetriableException) {
```

Comment:
Can we add a comment explaining why we do this? And nit: there should be a space after `if`.

Reply:
Added context to the change